letsencrypt/cli.py

```diff
@@ -159,7 +163,34 @@ def _init_le_client(args, config, authenticator, installer):
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
 
-def run(args, config, plugins):
+def _find_duplicative_certs(domains, config, renew_config):
+    """Find existing certs that duplicate the request."""
+
+    identical_names_cert, subset_names_cert = None, None
+
+    configs_dir = renew_config.renewal_configs_dir
+    for renewal_file in os.listdir(configs_dir):
+        full_path = os.path.join(configs_dir, renewal_file)
+        rc_config = configobj.ConfigObj(renew_config.renewer_config_file)
+        rc_config.merge(configobj.ConfigObj(full_path))
+        rc_config.filename = full_path
```

Comment:
code for initializing renewer configuration seems to follow the same pattern as in other parts of the source code. refactor?

```python
def load_renewer_config(renew_config, path):
    rc_config = configobj.ConfigObj(renew_config.renewer_config_file)
    rc_config.merge(configobj.ConfigObj(full_path))
    rc_config.filename = full_path
    return rc_config
```